### PR TITLE
Notmuch 0.24

### PR DESCRIPTION
**Pull request by benjaminfrank**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by benjaminfrank** [comment]
Sorry, didn't know that deleting a branch automatically closes an pull request

File: Formula/notmuch.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Notmuch < Formula
   desc "Thread-based email index, search, and tagging"
   homepage "https://notmuchmail.org"
-  url "https://notmuchmail.org/releases/notmuch-0.23.7.tar.gz"
-  sha256 "f11bb10d71945f6c3f16d23117afc70810aa485878e66bb4bf43cc3f08038913"
+  url "https://notmuchmail.org/releases/notmuch-0.24.tar.gz"
```
Comment (fxcoudert) [inline Formula/notmuch.rb]: Shouldn't the patch be removed?

Comment (dunn) [inline Formula/notmuch.rb]: Has it been merged? It hadn't the last time I checked.

Comment (fxcoudert) [inline Formula/notmuch.rb]: If upstream doesn't accept the patch, we shouldn't be carrying it anyway, as I understand our policy.

**Comment by benjaminfrank** [comment]
The patch isn't merged yet.
I think they're awaiting a reply from the person who did the python bindings but either he went missing or forgot about it.

**Comment by fxcoudert** [comment]
Thanks for the pull request. I've merged it, but in the long term we would like upstream to include the fix or we should drop the python bindings if they have stopped working and upstream doesn't care.

**Pull request closed without merging**